sceptre/config/reader.py

```diff
@@ -195,7 +204,7 @@ def construct_stacks(self):
 
     def read(self, rel_path, base_config=None):
         """
-        Reads in configuration from yaml files within the Sceptre project
+        Reads in configuration from a YAML file within the Sceptre project
```

Comment:
It can technically read multiple files in the hierarchy, since it uses `_recursive_read`.